projects/npm-tools/packages/npm-scripts/src/utils/createFederationConfig.js

```diff
@@ -189,6 +196,56 @@ module.exports = async function () {
 	remotes = remotes || [];
 	shared = shared || [];
 
+	// Prevent webpack from choking on scopes
+
+	const libraryName = makeNamespace(name);
+
+	const federationPluginConfig = {
+		exposes: {
+			...transformExposes(exposes, build.input),
+			'.': mainFilePath,
+		},
+		filename: 'container.js',
+		library: {
+			name: `window[Symbol.for("__LIFERAY_WEBPACK_CONTAINERS__")]["${libraryName}"]`,
+			type: 'assign',
+		},
+		name,
+		remoteType: 'script',
+		remotes: [...DEFAULT_REMOTES, ...remotes].reduce((remotes, remote) => {
+			let webContextPath;
+
+			if (typeof remote === 'string') {
+				webContextPath = remote;
+			} else {
+				webContextPath = remote.webContextPath;
+				remote = remote.name;
+			}
+
+			// Prevent webpack from choking on scopes
+
+			const libraryName = makeNamespace(name);
```

Comment:
Oops, this is wrong. Something went wrong during the rebase :-(. It should say `remote` instead of `name`.

Reply:
Fixed.